identity/handler.go

```diff
@@ -175,11 +175,12 @@ type listIdentitiesParameters struct {
 //	Responses:
 //	  200: listIdentities
 //	  default: errorGeneric
-func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+func (h *Handler) list(w http.ResponseWriter, r *http.Request, p httprouter.Params) {
```

Comment:
```suggestion
func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```

`p` is unused IIRC.

Reply:
Yea, this was accidental change from my first experiments. I've reverted that.